test/test.test.js

```diff
@@ -2,16 +2,17 @@
 
 const test = require('tap').test
 const Fastify = require('fastify')
-const rawJwt = require('jsonwebtoken')
+const { createSigner } = require('fast-jwt')
 const jwt = require('../jwt')
 
 const helper = require('./helper')
 
-const passphrase = 'super secret passphrase'
+// TODO: Uncomment when (if) fast-jwt adds suport for encoded private keys (passphrase protected)
+// const passphrase = 'super secret passphrase'
+// const { privateKey: privateKeyProtected, publicKey: publicKeyProtected } = helper.generateKeyPairProtected(passphrase)
+// const { privateKey: privateKeyProtectedECDSA, publicKey: publicKeyProtectedECDSA } = helper.generateKeyPairECDSAProtected(passphrase)
```

Comment:
Now that https://github.com/nearform/fast-jwt/pull/117 has been merged we can uncomment this.

Reply:
Done